SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -835,8 +834,8 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 	if (IsServer())
 	{
 		if (GlobalStateManager != nullptr && !GlobalStateManager->GetCanBeginPlay()
-			&& StaticComponentView->HasAuthority(GlobalStateManager->GlobalStateManagerEntityId,
-												 SpatialConstants::GDK_KNOWN_ENTITY_AUTH_COMPONENT_SET_ID))
+			&& Connection->GetCoordinator().HasAuthority(GlobalStateManager->GlobalStateManagerEntityId,
```

Comment:
Wonder if this should be bespoke functionality on the well-known-entities system?

Reply:
the has auth check? or the whole block? The block probably should, actually. Will move

Reply:
The `DoYouHaveAuthorityOverGSM` function

Reply:
moved the ifserver part into well known entity system